r-package/grf/R/causal_forest.R

```diff
@@ -257,8 +258,21 @@ causal_forest <- function(X, Y, W,
 #'                          (for confidence intervals).
 #' @param ... Additional arguments (currently ignored).
 #'
-#' @return Vector of predictions, along with (optional) variance estimates.
-#'
+#' @return Vector of predictions, along with estimates of the error and
+#'         (optionally) its variance estimates. Column 'predictions' contains estimates
+#'         of the conditional average treatent effect (CATE). The square-root of
+#'         column 'variance.estimates' is the standard error of CATE.
+#'         For out-of-bag estimates, we also output the following error measures.
+#'         First, column 'debiased.error' contains estimates of the 'R-loss' criterion, 
+#          a quantity that is related to the true (infeasible) mean-squared error 
+#'         (See Nie and Wager 2017 for a justification). Second, column 'excess.error' 
+#'         contains jackknife estimates of the Monte-carlo error (Wager, Hastie, Efron 2014), 
+#'         a measure of how unstable estimates are if we grow forests of the same size
+#'         on the same data set. The sum of 'debiased.error' and 'excess.error' is the raw error
+#'         attained by the current forest, and 'debiased.error' alone is an estimate of the error
+#'         attained by a forest with an infinite number of trees. We recommend that users grow 
+#'         enough forests to make the excess.error' negligible. 
```

Comment:
This should be 'excess.error' (missing quote).

Reply:
Ugh. I don't understand how I can be so sensitive about other people's typos and so blind to my own.